app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class UserMailer < ApplicationMailer
```

Comment:
mail should be deliver to user's email address in actual.
Read [this](http://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration)